src/components/AddReservation.jsx

```diff
@@ -0,0 +1,89 @@
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { fetchBranches } from '../redux/branchesSlice';
+import { fetchRooms } from '../redux/branchRoomSlice';
+import { saveReservation } from '../redux/saveReservationSlice';
+import { clearReservation } from '../redux/addReservationSlice';
+import { fetchReservations } from '../redux/reservationsSlice';
+import '../styles/Home.scss';
+import Card from './Card';
+import '../styles/Reservations.css';
+import '../styles/AddReservation.scss';
+
+const AddReservation = () => {
```

Comment:
- [ ] For some reason, I can't add new rooms because it brings me to a blank page:

![image](https://github.com/hernandanielzamora/Capstone-Front-End/assets/57421823/1acce410-13ab-4fdf-896e-80d7a76012c4)

If this is a bug, kindly fix it so your admin users are able to add rooms. 👍